test/integration/targets/vmware_guest/tasks/customization_hostname_validation.yml

```diff
@@ -0,0 +1,121 @@
+# Test code for the vmware_guest module.
+# Copyright: (c) 2019, Diego Morales <dgmorales@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Wait for Flask controller to come up online
```

Comment:
Hi @dgmorales dgmorales,

We are refactoring the vmware test-suite to keep it simple and be able to run it on a regular VMware environment.
The test roles should not interact with vcsim themselves anymore. You just need instead to import the `prepare_vmware_tests` role with the correct parameters. In addition, it's also important that you clean the resources that you create.

Example: https://github.com/ansible/ansible/blob/devel/test/integration/targets/vmware_guest_find/tasks/main.yml